package.json

```diff
@@ -5,7 +5,9 @@
   "main": "server/index.js",
   "private": true,
   "scripts": {
-    "dev": "docker-compose up"
+    "dev": "docker-compose up",
+    "test": "yarn workspaces run test",
+    "testAll": "docker-compose -f docker-compose.yml -f docker-compose.test.yml up"
```

Comment:
so this is for running test in a hotloaded way -- every time you change a file? Where are `yarn test` just runs everything once?

Reply:
Yes, that is correct